source/colors.js

```diff
@@ -15,255 +15,354 @@ module.exports = {
-        { tone: '30', value: '#5aaafa' },
-        { tone: '40', value: '#5596e6' },
-        { tone: '50', value: '#4178be' },
-        { tone: '60', value: '#325c80' },
-        { tone: '70', value: '#264a60' },
-        { tone: '80', value: '#1d3649' },
-        { tone: '90', value: '#152935' },
-        { tone: '100', value: '#010205' }
+        { tone: 'core', value: '#2d74da' },
+        { tone: '1', value: '#e1ebf7' },
+        { tone: '10', value: '#c8daf4' },
+        { tone: '20', value: '#a8c0f3' },
+        { tone: '30', value: '#79a6f6' },
+        { tone: '40', value: '#5392ff' },
+        { tone: '50', value: '#2d74da' },
+        { tone: '60', value: '#1f57a4' },
+        { tone: '70', value: '#25467a' },
+        { tone: '80', value: '#1d3458' },
+        { tone: '90', value: '#19273c' }
       ]
     },
     {
-      name: 'green',
+      name: 'cerulean',
+      synonyms: 'warm blue',
       values: [
-        { tone: 'core', value: '#4b8400' },
-        { tone: '10', value: '#c8f08f' },
-        { tone: '20', value: '#b4e051' },
-        { tone: '30', value: '#8cd211' },
-        { tone: '40', value: '#5aa700' },
-        { tone: '50', value: '#4b8400' },
-        { tone: '60', value: '#2d660a' },
-        { tone: '70', value: '#144d14' },
-        { tone: '80', value: '#0a3c02' },
-        { tone: '90', value: '#0c2808' },
-        { tone: '100', value: '#010200' }
+        { tone: 'core', value: '#009de2' },
+        { tone: '1', value: '#deedf7' },
+        { tone: '10', value: '#c2dbf4' },
+        { tone: '20', value: '#95c4f3' },
+        { tone: '30', value: '#56acf2' },
+        { tone: '40', value: '#009bef' },
+        { tone: '50', value: '#047cc0' },
+        { tone: '60', value: '#175d8d' },
+        { tone: '70', value: '#1c496d' },
+        { tone: '80', value: '#1d364d' },
+        { tone: '90', value: '#1b2834' }
+      ]
+    },
+    {
+      name: 'aqua',
+      values: [
+        { tone: 'core', value: '#00b6cb' },
+        { tone: '1', value: '#d1f0f7' },
+        { tone: '10', value: '#a0e3f0' },
+        { tone: '20', value: '#71cddd' },
+        { tone: '30', value: '#00b6cb' },
+        { tone: '40', value: '#12a3b4' },
+        { tone: '50', value: '#188291' },
+        { tone: '60', value: '#17616b' },
+        { tone: '70', value: '#164d56' },
+        { tone: '80', value: '#13393e' },
+        { tone: '90', value: '#122a2e' }
       ]
     },
     {
       name: 'teal',
       values: [
-        { tone: 'core', value: '#008571' },
-        { tone: '10', value: '#a7fae6' },
-        { tone: '20', value: '#6eedd8' },
-        { tone: '30', value: '#41d6c3' },
-        { tone: '40', value: '#00b4a0' },
-        { tone: '50', value: '#008571' },
-        { tone: '60', value: '#006d5d' },
-        { tone: '70', value: '#005448' },
-        { tone: '80', value: '#003c32' },
-        { tone: '90', value: '#012b22' },
-        { tone: '100', value: '#000202' }
+        { tone: 'core', value: '#00A78F' },
```

Comment:
Lower-case all `teal` values for consistency and efficiency (http://www.websiteoptimization.com/speed/tweak/lowercase/)?